README.md

```diff
@@ -232,6 +232,10 @@ lsp-dart supports running Flutter and Dart commands as following:
 
 :small_blue_diamond: Try to set the `lsp-dart-sdk-dir` to the Dart SDK dir installation or if you are using Flutter, `<your-flutter-dir>/bin/cache/dart-sdk/`.
 
+:grey_question: `dap-debug` couldn't run the app on advices or white screen.
+
+:small_blue_diamond: Try to set `export NO_PROXY=localhost,127.0.0.1` into your `.bash_profile` file.
```

Comment:
Maybe link that issue from dart code?

Reply:
I think that if we have already known how to solve a problem, we should just tell users as terse as we could lest they haven't got enough patience to read.